src/main/java/pl/poznan/put/cs/scenariochecker/transformations/ScenarioHelper.java

```diff
@@ -14,4 +16,15 @@ private ScenarioHelper() {
     public static boolean isSpecialStep(Step step) {
         return step.getContent().startsWith("IF") || step.getContent().startsWith("FOR EACH");
     }
-}
+
+    /**
+     * @param step, list of actors
+     * @return true when step is step without Actor's
+     */
+    public static boolean isNoActorStep(Step step, List<String> actors) {
+        for (String actor : actors) {
```

Comment:
I think you should also take into account system actor when checking if a certain step doesn't start with actor's name.